gradio/documentation.py

```diff
@@ -58,8 +58,8 @@ def document_fn(fn: Callable) -> Tuple[str, List[Dict], Dict, Optional[str]]:
             if mode == "description":
                 description.append(line if line.strip() else "<br>")
                 continue
-            assert line.startswith(
-                "    "
+            assert (
+                line.startswith("    ") or line.strip() == ""
```

Comment:
Can you explain this change?

Reply:
^ correct, to allow empty lines in docstring, used in Example section where example code may have empty line